src/objects.js

```diff
@@ -1,41 +1,51 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  return person.age > 65;
 };
 
 const getAges = people => {
-  // your code here
+  return people.map(person => person.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(person => person.name === name);
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter(car => car.manufacturer === 'Honda');
 };
 
 const averageAge = people => {
-  // your code here
+  const calculateAverage = people.reduce((acc, person) => acc + person.age, 0) / people.length;
+  return calculateAverage;
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age,
+    introduce: function introduce(person) {
+      return `Hi ${person}, my name is ${this.name} and I am ${this.age}!`;
+    }
+  };
```

Comment:
same as before. The key's name are the same as the variable and you can use it as shorthand:

````
return {
     name, 
     age,
     introduce(person){
          return `Hi ${person}, my name is ${this.name} and I am ${this.age}!`;
}